deployment/common/changeset/transfer_link_token.go

```diff
@@ -0,0 +1,71 @@
+package changeset
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	gethtypes "github.com/ethereum/go-ethereum/core/types"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+)
+
+var _ deployment.ChangeSet[TransferLinkTokenBatchConfig] = TransferLinkTokenBatch
+
+type TokenTransferer interface {
+	Transfer(opts *bind.TransactOpts, to common.Address, amount *big.Int) (*gethtypes.Transaction, error)
+}
+
+type TransferLinkTokenConfig struct {
+	To         common.Address
+	Amount     *big.Int
+	Transferer TokenTransferer
```

Comment:
I don't think this should be part of the config/input. Users shouldn't want to think how the tx is formed, only that is sent correctly

Reply:
Yes, I think the input of the changeset should simply be `to` and `amount`. How the transaction is formed and sent should be abstracted away from the user

Reply:
interesting, but how do we perform the transfer if we dont pass the contract in? I was folloing the accept ownership pattern [here](https://github.com/smartcontractkit/chainlink/blob/fe15cc55e77a88a2d3d8d0f12ac4f1998d351961/deployment/common/changeset/accept_ownership.go#L33-L33), or is there a better way ?